src/plugins/definitions/ai-insights.tsx

```diff
@@ -24,6 +26,42 @@ const createAiInsightsPlugin = (): Plugin => ({
         },
       }
     );
+    scope.appendSlot(
+      'executionDrawerBanner',
+      <MessagePanel
+        buttons={[
+          {
+            type: 'secondary',
+            text: 'Learn more',
+            isLink: true,
+            linkConfig: {
+              href: 'https://testkube.io/get-started',
+              target: '_blank',
+            },
+          },
+          {
+            type: 'primary',
+            text: 'Get AI Hints for this test',
+            onClick: () => {
+              scope.getState<AiBannerInterface>('aiExecutionBanner')?.onAccept();
+            },
+          },
+        ]}
+        onClose={() => {
+          scope.getState<AiBannerInterface>('aiExecutionBanner')?.onClose();
+        }}
+        isClosable
+        type="default"
+        title="🎉 Get AI support on failing tests! 🎉"
+        description="Try our AI Hints to improve your tests and get support to debug them more efficiently."
+      />,
+      {
+        visible: () =>
+          localStorage.getItem(config.isAiBannerVisible) === 'true' &&
+          scope.getState<TestExecutionTabsInterface>('testExecutionTabs')?.execution?.executionResult?.status ===
```

Comment:
Logic for checking failed status could be split to a local function and reused for both `visible` metadata in the plugin